src/layers/legacy/medCoreLegacy/gui/lookUpTables/medClutEditor.h

```diff
@@ -30,6 +30,7 @@ class medClutEditorVertexPrivate;
 class MEDCORELEGACY_EXPORT medClutEditorVertex : public QObject, public QGraphicsItem
 {
     Q_OBJECT
+    Q_INTERFACES(QGraphicsItem)
```

Comment:
Fo my cruiosity can you explain the role of Q_INTERFACES

Reply:
I noticed that we had warning about the missing Q_INTERFACES. It is used to create QObject classes to list which qt class it inherits (https://doc.qt.io/qt-5/qobject.html#Q_INTERFACES)